feature/savings-account/src/commonMain/kotlin/org/mifos/mobile/feature/savingsaccount/SavingsAccountScreen.kt

```diff
@@ -0,0 +1,329 @@
+
+        null -> Unit
+    }
+}
+
+@Composable
+internal fun SavingsAccountContent(
+    state: SavingsAccountState,
+    onAction: (SavingsAccountAction) -> Unit,
+) {
+    val pullToRefreshState = rememberMifosPullToRefreshState(
+        isEnabled = true,
+        isRefreshing = state.isRefreshing ?: false,
+        onRefresh = {
+            onAction(SavingsAccountAction.Refresh)
+        },
+    )
+
+    MifosElevatedScaffold(
+        onNavigateBack = { onAction(SavingsAccountAction.OnNavigateBack) },
+        topBarTitle = stringResource(Res.string.feature_savings_account_topbar_title),
+        pullToRefreshState = pullToRefreshState,
+        bottomBar = {
+            Surface {
+                MifosPoweredCard(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .navigationBarsPadding(),
+                )
+            }
+        },
+    ) {
+        Column(
+            modifier = Modifier
+                .fillMaxSize()
+                .padding(DesignToken.padding.large),
+        ) {
+            Spacer(modifier = Modifier.height(DesignToken.spacing.large))
+
+            MifosDashboardCard(
+                isSingleLine = true,
+                savingsAccount = Res.string.feature_savings_account_dashboard,
+                savingsAmount = state.totalSavingAmount,
+                isVisible = state.isAmountVisible,
+                currency = state.currency,
+                onVisibilityToggle = { onAction(SavingsAccountAction.ToggleAmountVisible) },
+            )
+
+            Spacer(modifier = Modifier.height(DesignToken.spacing.largeIncreased))
+
+            Row(
+                modifier = Modifier.fillMaxWidth(),
+                verticalAlignment = Alignment.CenterVertically,
+                horizontalArrangement = Arrangement.SpaceBetween,
+            ) {
+                Column {
+                    Text(
+                        text = stringResource(Res.string.feature_savings_account),
+                        style = MifosTypography.titleMediumEmphasized,
+                        color = MaterialTheme.colorScheme.onBackground,
+                    )
+                    Text(
+                        text = stringResource(
+                            Res.string.feature_savings_account_items,
+                            state.items ?: 0,
+                        ),
+                        style = MifosTypography.labelMedium,
+                        color = MaterialTheme.colorScheme.secondary,
+                    )
+                }
+
+                Row(
+                    horizontalArrangement = Arrangement.spacedBy(DesignToken.spacing.largeIncreased),
+                ) {
+                    Icon(
+                        modifier = Modifier
+                            .clickable {}
+                            .size(20.dp),
+                        imageVector = MifosIcons.SearchNew,
+                        contentDescription = "search",
```

Comment:
use localized strings to support accessibility and internationalization. do it for strings that u harcoded